app/blockchain/fabric-samples/ci/scripts/run-test-network-sbe.sh

```
@@ -0,0 +1,34 @@
+set -euo pipefail
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)